pkg/spec/spec.go

```diff
@@ -553,6 +559,20 @@ func addRlimits(config *CreateConfig, g *generate.Generator) error {
 	)
 
 	for _, u := range config.Resources.Ulimit {
+		if u == "host" {
```

Comment:
Is there really no way to tell runc "just don't change rlimits"?

Reply:
it seems I've unnecessarily complicated things.  Looks like it is enough to not specify any limits block and that won't change the rlimits.  I'll do some more tests and open a PR

Reply:
much simpler version: https://github.com/containers/libpod/pull/3583